### ISSUE TITLE
Plugin per row shortcut

### ISSUE THREAD

**Issue by lhoguin** [zhongwencool/observer_cli#78]
I would like to have my plugin be able to set a shortcut for each row in the sheet, similar to how we can choose processes in the Home view. In my case I could select a RabbitMQ queue I want to get more information about. It would be good if I could also link this detailed queue view to its detailed process view.

What do you think?

**Comment by zhongwencool** [comment]
Yes! I plan to do this a long time ago, I can put it into the next version.

Also Plan to Enhance the application view: 
make application sort by reduction,memory,process count.

**Comment by lhoguin** [comment]
I can test your changes before you produce a release and provide feedback if you want (during the work week).
